templates/fargate.cfn.yml

```diff
@@ -291,7 +299,7 @@ Resources:
               pre_build:
                   commands:
                   - printenv
-                  - TAG="$ENVIRONMENT_NAME.$(date +%Y-%m-%d).$(echo $CODEBUILD_RESOLVED_SOURCE_VERSION | head -c 8)"
+                  - TAG="$REPOSITORY_NAME.$REPOSITORY_BRANCH.$ENVIRONMENT_NAME.$(date +%Y-%m-%d.%H.%M.%S).$(echo $CODEBUILD_RESOLVED_SOURCE_VERSION | head -c 8)"
```

Comment:
This is a lot of specificity for a Docker container image tag. I'm not sure what you're envisioning where this amount of detail is really required. An application will typically have a single source repository, that repository should have a single mainline branch, and aspects of the build such as the timestamp that the tag was generated seem superfluous.

Reply:
We can discuss tomorrow, but since I am not specifying a name for the ECR repository, this clears things up a bit in the console. Currently, I am not generating a new repository per application because this seems unnecessary. I am not religious about this, so we can chat about it on our call.

Reply:
I don't have a strong opinion here. Just stood out to me.